model_api/python/model_api/tilers/detection.py

```diff
@@ -184,7 +185,8 @@ def _merge_saliency_maps(self, saliency_maps, shape, tiles_coords):
         image_map_w = int(image_w * ratio[1])
         merged_map = np.zeros((num_classes, image_map_h, image_map_w))
 
-        for i, saliency_map in enumerate(saliency_maps[1:], 1):
+        saliency_maps, start_idx = (saliency_maps[1:], 1) if self.tile_with_full_img else (saliency_maps, 0)
+        for i, saliency_map in enumerate(saliency_maps, start_idx):
```

Comment:
```suggestion
        start_idx = 1 if self.tile_with_full_img else 0
        for i, saliency_map in enumerate(saliency_maps[start_idx:]):
```
We can simplify that since `i` is just an index to iterate through list of saliency maps without any additional meaning